include/filewriter/summarywriter.hpp

```diff
@@ -0,0 +1,43 @@
+/**
+ * @file filewriter.hpp
+ * @author Jeffin Sam
+ */
+#ifndef MLBOARD_SUMMARY_WRITER_HPP
+#define MLBOARD_SUMMARY_WRITER_HPP
+
+#include <mlboard/core.hpp>
+#include "filewriter.hpp"
+#include <proto/summary.pb.h>
+
+namespace mlboard {
+
+/**
+ * Class responsible to create a summary to be logged.
+ * 
+ * @tparam filewriter The filewriter object whould would convert it
+ *    into events and then log to a file. 
+ */
+template<typename filewriter>
+class SummaryWriter
+{
+ public:
+  /**
+  * A function to create a scaler summary
```

Comment:
A missing period at the end of the comment.

Reply:
Done